app/views/users/show_user_tweets.erb

```diff
@@ -0,0 +1,6 @@
+<h2>Here's all your tweets, <%=@user.username%></h2>
+
+<% binding.pry %>
+  <%@user.tweets.each do |tweet|%>
```

Comment:
we should abstract these database calls in the controller. The view is to display the logic the controller gives us. 
```
@user_tweets = current_user.tweets
```